### PR TITLE
Menu Project Gold

### PR DESCRIPTION
**Pull request by Brennaleker**
Random menu generator with breakfast, lunch and dinner requirements completed.
Thank you for taking the time to review my code :)

### CODE REVIEW

File: menu_gold.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#setting up arrays for each of the variables
+ingredient = ["baby banana", "Runts banana", "corn ball", "plantain", "classic Dole banana", "manzano banana", "burro banana", "goldfinger banana", "ladyfinger banana", "golden beauty banana"]
+prepared = ["seared", "fried", "braised", "corn-balled", "dipped", "grilled", "steamed", "sauteed", "smoked", "baked"]
+adjective = ["chocolatey", "cheesy", "candied", "salted", "spicy", "sugar-coated", "firey", "crunchy", "caramelized", "breaded"]
+#creating a variable to track the smallest array length so that I can alert the user to how many unique menu items they can create
+item_count = [ingredient.length, adjective.length, prepared.length].min
```
Comment (GreyKn) [inline menu_gold.rb:6]: Nice use of `.min` to determine your loop count. :+1:

File: menu_gold.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#setting up arrays for each of the variables
+ingredient = ["baby banana", "Runts banana", "corn ball", "plantain", "classic Dole banana", "manzano banana", "burro banana", "goldfinger banana", "ladyfinger banana", "golden beauty banana"]
+prepared = ["seared", "fried", "braised", "corn-balled", "dipped", "grilled", "steamed", "sauteed", "smoked", "baked"]
+adjective = ["chocolatey", "cheesy", "candied", "salted", "spicy", "sugar-coated", "firey", "crunchy", "caramelized", "breaded"]
```
Comment (GreyKn) [inline menu_gold.rb:4]: I'm impressed that you listed all your array elements consistently (comma followed by space, all double quotes, no dangling elements). As someone with borderline ocd that reviews code all day, I appreciate it :tada:

File: menu_gold.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+#will insure that none of the array itemsare use more than once
+elsif user_count.to_i <= item_count
+  #setting up a counter to track how many times the block has run
+  count = 0
+  #in the silver level user_count was menu_count
+  user_count = user_count.to_i
+    until
+    #the code will run until the number of times the block has run(count)
+    #equals the number of menu items requested(user_count)
+    count == user_count
```
Comment (GreyKn) [inline menu_gold.rb:38]: If you place the control conditional on the same line as `until`, it'll be easier to read.

```
until count == user_count
...
end
```

File: menu_gold.rb [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    Caw! Ca-caw! Ca-caw! Ca-caw!\n"
+
+#if the user selects a number less than or equal to the number of available array items, the program
+#will insure that none of the array itemsare use more than once
+elsif user_count.to_i <= item_count
+  #setting up a counter to track how many times the block has run
+  count = 0
+  #in the silver level user_count was menu_count
+  user_count = user_count.to_i
+    until
```
Comment (GreyKn) [inline menu_gold.rb:35]: I think your indenting is slightly off here

File: menu_gold.rb [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  user_count = gets.chomp
+end
+
+#special message if the user states that they do not want to see any menu items
+if user_count == "0"
+    puts "\nChicken!!!\n
+    Caw! Ca-caw! Ca-caw! Ca-caw!\n"
+
+#if the user selects a number less than or equal to the number of available array items, the program
+#will insure that none of the array itemsare use more than once
```
Comment (GreyKn) [inline menu_gold.rb:29]: So if the user inputs a number greater than the number of available items, they can get repeats?

**Comment by GreyKn** [comment]
Hi Brenna,

Great job with this assignment. I think you fulfilled all the necessary requirements for Dinner level :100: 
The only general feedback I have for you is be a little more consistent with how you're indenting your code. It makes a huge difference in how readable everything is.

Let me know if you have any questions.

**Comment by Brennaleker** [comment]
Hi,
I will try to be more consistent with my indentation in future projects. I've noticed that when I am consistent not only is it easier to read but it also cuts down on my terrible tendency to forget an end or two in my blocks.  

Thank you for taking the time to review my code. Your input is very much appreciated.

**Pull request merged**